docker/pyramid/Dockerfile

```diff
@@ -56,12 +60,22 @@ COPY --chown=igvfd:igvfd src/igvfd/schema_formats.py src/igvfd/schema_formats.py
 
 COPY --chown=igvfd:igvfd src/igvfd/ontology.py src/igvfd/ontology.py
 
+COPY --chown=igvfd:igvfd src/igvfd/audit/* src/igvfd/audit/
+
 COPY --chown=igvfd:igvfd assets/ontology.json.gz assets/ontology.json.gz
 
 COPY --chown=igvfd:igvfd src/igvfd/commands/make_reference_db.py src/igvfd/commands/make_reference_db.py
 
+COPY --chown=igvfd:igvfd src/igvfd/commands/make_audit_docstring_json.py src/igvfd/commands/make_audit_docstring_json.py
+
 RUN make-reference-db
 
+RUN touch /static/json/auditdoc.json
+
+RUN chown -R igvfd:igvfd /static/json/auditdoc.json
+
+RUN make-audit-docstring-json
```

Comment:
I think run this after the full app has been copied over, then dump it in the /igvfd/src/static/json folder?

Reply:
That directory gets masked by docker compose volume.

Reply:
changed according to your advice below.